### DISCUSSION TITLE
SSH keys

### DISCUSSION THREAD

**Discussion by Aishaakin** [community/community#42288] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

HI everyone i am having a problem make a SSH keys for git but i don't know how to do it, please can someone show me a resource or easy video on how to make a SSH key on github. I am new to programming and i have checked other ways to do it like researching on resources or videos but i am still confuse. Please help

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @Aishaakin 

There's a section in the GitHub docs explaining this, here's the link: [https://docs.github.com/pt/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account](https://docs.github.com/pt/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)

**Reply by Aishaakin** [reply]
> Hi @Aishaakin 
> 
> There's a section in the GitHub docs explaining this, here's the link: [https://docs.github.com/pt/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account](https://docs.github.com/pt/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)

Thank you
